pygeoapi-config.yml

```diff
@@ -125,3 +125,33 @@ datasets:
             name: GeoJSON
             data: tests/data/ne_110m_lakes.geojson
             id_field: null  # null indicates use feature enumeration
+     
+    countries:
+        type: Polygon
+        id_field: ogc_fid  # null indicates use feature enumeration
+        title: Countries in the world
+        description: Countries of the world
+        keywords:
+            - countries
+            - natural eart
+        crs:
+            - CRS84
+        links:
+            - type: text/html
+              description: information
+              url: http://www.naturalearthdata.com/
+            - type: text/html
+              description: download
+              url: http://www.naturalearthdata.com/
+        extents:
+            spatial:
+                bbox: [-180,-90,180,90]
+            temporal:
+                begin: None
+                end: now  # or empty
+        provider: 
+            name: Sqlite
+            data: tests/data/ne_110m_admin_0_countries.sqlite
+            id_field: ogc_fid
+            table: ne_110m_admin_0_countries
```

Comment:
Looks like we have also added `tests/data/ne_110m_lakes.sqlite`? Do we need this SQLite file or can we remove?

Reply:
As well, we will probably want a README.md in `test/data` with some info on data sources.

Reply:
Ignore for now, I'll add a README.md in another PR.

Reply:
The ne_110m_lakes.sqlite is no  longer needed will remove